astropy/tests/disable_internet.py

```diff
@@ -55,8 +55,9 @@ def new_function(*args, **kwargs):
             valid_hosts = ('localhost', '127.0.0.1')
 
         if allow_astropy_data:
-            data_astropy_org_ip = socket.gethostbyname('data.astropy.org')
-            valid_hosts += ('data.astropy.org', 'astropy.stsci.edu', data_astropy_org_ip)
+            for valid_host in ('data.astropy.org', 'astropy.stsci.edu'):
```

Comment:
Huh. I am not aware that we have `astropy.stsci.edu`. Is this a planned addition for 1.3?